### DISCUSSION TITLE
Your Xcode (12.5) is too outdated. Please update to Xcode 13.0...

### DISCUSSION THREAD

**Discussion by LER0ever** [Homebrew/discussions#1798] [Everyday usage] [answered]
Homebrew complains about Xcode (latest version) being too old, and needs Xcode 13, which is not yet released.

```text
Error: Your Xcode (12.5) is too outdated.
Please update to Xcode 13.0 (or delete it).
Xcode can be updated from:
  https://developer.apple.com/download/more/
```

This happens when performing a regular `brew update && brew install pandoc`, after which it tries to update the dependents of a package, and printed this error during a cask install (`d12frosted/emacs-plus/emacs-plus@27`).

The system information is also attached in the screenshot, (M1 Macbook Air, macOS Monterey beta 2).

![image](https://user-images.githubusercontent.com/10568668/124880530-f7fb5e80-e000-11eb-9cda-0415adb1efe3.png)

**Comment by SMillerDev** [comment, marked as answer]
> Homebrew complains about Xcode (latest version) being too old, and needs Xcode 13, which is not yet released.

But it's the latest version for your OS (which is also not released yet)

**Reply by LER0ever** [reply]
Think I found the OS <-> Xcode mapping here: https://github.com/Homebrew/brew/blob/47d5150c333784c956db25c3d8ff8ffdd616bb68/Library/Homebrew/os/mac/xcode.rb#L25. I'll try to do a local ad-hoc fix then.

> But it's the latest version for your OS (which is also not released yet)

True 🤣
